### PR TITLE
Add/Update troubleshooting/logs docs

### PR DESCRIPTION
**Pull request by kcoriordan**
## What this PR does / why we need it

 - [Troubleshoot Clusters with Tanzu Diagnostics](https://tanzucommunityedition.io/docs/v0.11/tanzu-diagnostics/) & [Troubleshoot Cluster Bootstrapping](https://tanzucommunityedition.io/docs/v0.11/tsg-bootstrap/) have largely the same content. This PR combines the common content into [Troubleshoot Clusters with Tanzu Diagnostics](https://tanzucommunityedition.io/docs/v0.11/tanzu-diagnostics/) and deletes [Troubleshoot Cluster Bootstrapping](https://tanzucommunityedition.io/docs/v0.11/tsg-bootstrap/)  - this doesn't introduce new content - it's just restructure

- Creates a new topic with some of the content from [Troubleshoot Cluster Bootstrapping](https://tanzucommunityedition.io/docs/v0.11/tsg-bootstrap/) - Troubleshoot Cluster Deployment Manually - this doesn't introduce new content - it's just restructure

- Adds a new topic to address #3253 

- Fixes the right nav headings in [Cluster Debugging Tips](https://tanzucommunityedition.io/docs/v0.11/tanzu-debugging-tips/) so that there is one nav item per tip. 

- Expands bootstrap glossary term with some content that was in [Troubleshoot Cluster Bootstrapping](https://tanzucommunityedition.io/docs/v0.11/tsg-bootstrap/) 

## Which issue(s) this PR fixes

Fixes: #https://github.com/vmware-tanzu/community-edition/issues/3253

### CODE REVIEW

**Comment by kcoriordan** [comment]
@jakerobinson

File: docs/site/content/docs/tanzu-diagnostics.md [hunk 1]
```diff
@@ -16,7 +18,20 @@ Other requirements include:
 * Access to a management cluster and its managed workload clusters (if needed)
 * Access to a local unmanaged cluster
 
-## Collecting diagnostics
+### Before you Begin
+
+If you want to debug the bootstrap cluster, first, determine the name of the bootstrap cluster:
+
+```sh
+kind get clusters
```
Comment (stmcginnis) [inline docs/site/content/docs/tanzu-diagnostics.md:26]: We don't require the user to install `kind` as part of a normal install. I wonder if we should do something like `kubectl config get-contexts` or `docker ps` to find the cluster.

**Review by stmcginnis — COMMENTED** [review]
There is some issue causing the netlify build to fail. Otherwise looks good.

File: docs/site/content/docs/tanzu-diagnostics.md (hunk 1 above)
Comment (kcoriordan) [reply docs/site/content/docs/tanzu-diagnostics.md:26]: This was added for #https://github.com/vmware-tanzu/community-edition/issues/1716
I think finding the kind cluster name here is in the context of a failed management cluster deploy where the kind cluster was installed but the cluster failed to deploy and delete the kind cluster

The 2nd paragraph in the introduction to this topic explains this - and this content originally came from a video from @joshrosso :

>When debugging a failed management cluster deployment, the kind bootstrap cluster is the key to being able to introspect and understand what is happening. Any issues or errors that occur in the kind bootstrap cluster during bootstrapping will provide information about potential problems in the final management cluster on the target provider. If the bootstrap process is stuck or did not finish successfully, use the diagnostics plugin to collect logs and cluster information.

Comment (stmcginnis) [reply docs/site/content/docs/tanzu-diagnostics.md:26]: 1716 didn't add requiring `kind`. It looks like that added a way to look up the name without needing to resort to `kind`. I think we should stick with that so we are not telling end users to install yet another thing, just to look up a name.

File: docs/site/content/docs/glossary.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Same as packages (see below).
 
 ### Bootstrap
 
-The bootstrap (noun) machine is the laptop, host, or server on which you download and run the Tanzu CLI. This is where the initial bootstrapping (verb) of a management cluster occurs before it is pushed to the platform where it will run. You run `tanzu`, `kubectl` and other commands on the bootstrap machine.
+In the context of a managed cluster, the bootstrap (noun) machine is the laptop, host, or server on which you download and run the Tanzu CLI. This is where the initial bootstrapping (verb) of a managed or cluster occurs before it is pushed to the platform where it will run. In the process of deploying a management cluster, a bootstrap cluster is created on your local client machine. This is a [Kind](https://kind.sigs.k8s.io/) based cluster - a cluster in a container. This bootstrap cluster then creates a cluster on your specified provider. You run `tanzu`, `kubectl` and other commands on the bootstrap machine.
```
[the remark this answers is not in the document]
Comment (kcoriordan) [reply docs/site/content/docs/glossary.md]: cleaned this up a bit

File: docs/site/content/docs/tanzu-diagnostics.md (hunk 1 above)
Comment (jpmcb) [reply docs/site/content/docs/tanzu-diagnostics.md:34]: Will the context be automatically set to the kind cluster during bootstrapping?

If so, we can simplify it even further by suggesting the user inspect the current context with `kubectl config view --minify`. if not, they will have to look at all the contexts and grab the name from there.

---

+1 - I agree with Sean, i don't think the `kind` CLI is a requirement here and the same can be achieved easily with an existing pre-req, `kubeclt`.

Comment (kcoriordan) [reply docs/site/content/docs/tanzu-diagnostics.md:40]: OK thanks, I worked in both of those commands now

**Comment by jpmcb** [comment]
@kcoriordan - looks to be merge conflicts on this one. Otherwise, LGTM

**Comment by kcoriordan** [comment]
closing and replacing with #4379

**Pull request closed without merging**